addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -313,15 +313,15 @@ def test_tax_affect_base_2(self):
                 'Untaxed Amount': [
                     {
                         'tax_group_name': self.tax_group1.name,
-                        'tax_group_amount': 450,
-                        'tax_group_base_amount': 2300,
+                        'tax_group_amount': 420,
+                        'tax_group_base_amount': 2000,
```

Comment:
While this is correct, it would be better if we fix the tests [the same way we did in master](https://github.com/odoo/odoo/blob/master/addons/account/tests/test_invoice_tax_totals.py#L287), by adding explicit sequences to taxes. Will make your FWs easier :wink: